### PR TITLE
updating node operation introduction

### PR DESCRIPTION
**Pull request by vishalchangrani**
Closes: #???

## Description

______

For contributor use:

- [ ] Targeted PR against `master` branch
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Updated relevant documentation 
- [ ] Re-reviewed `Files changed` in the Github PR explorer
- [ ] Added appropriate labels

### CODE REVIEW

File: docs/content/node-operation/index.mdx [hunk 1]
```diff
@@ -4,7 +4,33 @@ sidebar_title: Introduction
...
 
-To run a node in the Flow network, the node must:
+
+## Why run a node?
+
+Running a node lets you participate in the network and contribute to the security and decentralization of the network.
+A node also gives you direct access to the evolving state of the network.
+Flow multirole architecture not it only makes it more scalable but also provides several node types that you as a node operator can pick and choose from.
+
+The different types of nodes are described [here](/node-operation/node-roles). As node operator, you can choose to run any of the different types of node that best fits your needs
```
Comment (muttoni) [inline docs/content/node-operation/index.mdx:14]: needs period

File: docs/content/node-operation/node-roles.mdx [hunk 2]
```diff
@@ -5,6 +5,10 @@ title: Node Roles
 Unlike most blockchains, not all Flow nodes are equal. Flow nodes all specialize and fulfill a specific role in the operation of the network.
 Collection, consensus, execution, verification and access nodes are all staked nodes while the observer node is not staked.
 
+## Observer :new:
+
+An observer node provides locally accessible, continuously updated, verified copy of the block data. It serves the Access API but unlike an access node, an observer node does not need to be staked, and **anyone** can run it without being added to the approved list of nodes.
```
Comment (muttoni) [inline docs/content/node-operation/node-roles.mdx:10]: Add a CTA linking to the page that explains how to run it

File: docs/content/node-operation/node-setup.mdx [hunk 3]
```diff
@@ -16,9 +16,12 @@ The hardware your Node will need varies depending on the role your Node will pla
 | **Execution**    | 64 cores  | 800 GB |  9 TB  | n2-highmem-128 |
 | **Verification** |  2 cores  | 16 GB  | 200 GB | n2-highmem-2   |
 | **Access**       |  16 cores | 64 GB  | 900 GB | n2-standard-16 |
+| **Observer**     |  2 cores  | 4 GB   | 300 GB | n2-highmem-16  |
```
Comment (muttoni) [inline docs/content/node-operation/node-setup.mdx:19]: just double checking: is n2-highmem-16 the right machine for an observer node? Looking at GCP, it seems n2-standard-4 (or similar) is a better fit. Seems strange that Observer has a more powerful machine that Access, right?

File: docs/content/node-operation/node-setup.mdx [hunk 4]
```diff
@@ -16,9 +16,12 @@ The hardware your Node will need varies depending on the role your Node will pla
...
 
 _Note: The above numbers represent our current best estimate for the state of the network. These will be actively updated as we continue benchmarking the network's performance._
 
+> For the observer node setup, follow this [guide](/node-operation/observer-node).
```
Comment (muttoni) [inline docs/content/node-operation/node-setup.mdx:23]: Rephrase to: "To run an Observer node, follow this [guide](/node-operation/observer-node)."

File: docs/content/node-operation/node-roles.mdx (hunk 2 above)
Comment (muttoni) [reply docs/content/node-operation/node-roles.mdx:10]: example: [Get started running an observer node](/node-operation/observer-node)

**Review by muttoni — CHANGES_REQUESTED** [review]

File: docs/content/node-operation/node-setup.mdx (hunk 3 above)
Comment (vishalchangrani) [reply docs/content/node-operation/node-setup.mdx:19]: ah good catch!! i think we overprovisioned our observer node and I blindly copied the instance type.

File: docs/content/node-operation/index.mdx [hunk 5]
```diff
@@ -4,7 +4,34 @@ sidebar_title: Introduction
...
+A node also gives you direct access to the evolving state of the network.
+Flow multirole architecture makes it more scalable and provides several node types that you as a node operator can pick and choose from.
+
+
+## Which node should you run?
+
+The different types of nodes are described [here](/node-operation/node-roles). As node operator, you can choose to run any of the different types of node that best fits your needs.
+
+### Observer node
+The observer node is one of the easiest node to spin up and is ideal for Dapps that need access to the latest block data available locally e.g a wallet application that needs to track the latest block ID and height.
```
Comment (10thfloor) [inline docs/content/node-operation/index.mdx:20]:
```suggestion
The observer node is one of the easiest nodes to spin up. Dapps interact with observer nodes when they need the latest block data locally eg. a wallet application that needs to track the latest block ID and height.
```

File: docs/content/node-operation/index.mdx [hunk 6]
```diff
@@ -4,7 +4,34 @@ sidebar_title: Introduction
...
+The observer node is not staked but still provides the same API as the access node.
+In addition to supporting dapps, an observer node can also be run by access node operators who want to scale their access nodes' endpoints geographically can spin up geographically dispersed observer nodes which can talk to their staked access nodes and to each other.
+To run an observer node, follow this [guide](/node-operation/observer-node).
```
Comment (10thfloor) [inline docs/content/node-operation/index.mdx:23]:
```suggestion
In addition to supporting dapps, an observer node can also be run by access node operators who want to scale their access nodes' endpoints. A user can spin up geographically dispersed observer nodes which can talk to their staked access nodes and to each other.
To run an observer node, follow this [guide](/node-operation/observer-node).
```

**Review by 10thfloor — CHANGES_REQUESTED** [review]

**Review by narangchirag — APPROVED** [review]

**Review by 10thfloor — APPROVED** [review]

**Pull request merged**